code-challenges.test.js

```diff
@@ -18,6 +18,31 @@
 
 // a) Create a test with expect statements for each of the variables provided.
 
+// The first step I want to do is create a function. 
+// The first step to creating a function is to declare a variable- example const.
+// Second step is to create a function name which allows us to call on the function.
+
+const numsGreat = (num) => { // call numsGreat with value greater than 1
+    if (num<2)
+        return num;
+    let arr = Array(num); // create an empty with a size of number passed in
+    arr[0]=1; // set first element to 1.
+    arr[1]=1; // set second element to 1.
+
+    // for 2 to num less 1 do the following
+    for (i=2;i<num;i++)
+    {
+        // current array element is equal the sum of the prior two
+        arr[i]=arr[i-1]+arr[i-2];
+    }
+    return arr; // return array
+}
+let arr2 = numsGreat(2);
```

Comment:
You've done a great job here with your logic but I haven't seen this syntax before. It works - so cool!